src/components/main.js

```diff
@@ -435,6 +435,7 @@ class Main extends React.Component {
         return this.setError({ msg: err, cb: this.refreshWallets })
       this.setError();
     })
+    this.fetchBtcData()
```

Comment:
Shouldn't this be inside the callback? Like right after `this.setError()`?

Reply:
Yep. It should.

I'm still having issues getting `blockchain.info` requests to go through. It's sending `429` status code responses every time. I'm looking into it.

Reply:
d801487

Reply:
That's super weird... okay this change is good